x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/ml_model_toasts.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { useComponentTemplatesContext } from '../../../component_templates/component_templates_context';
+
+export function MLModelNotificationToasts() {
```

Comment:
So this is a custom React use function now. We should call it: `useMlModalNotificationToasts` (or something similar). Starting with an upper case letter is reserved for React components, which this isn't anymore.